script.js

```diff
@@ -1,11 +1,18 @@
 // Create an array called `colors` that contains three colors.
-
+colors = ['green', 'blue', 'orange']
 // Log the last item in the array to the console.
-
+console.log(colors[2])
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (i=0; i<colors.length; i++) {
+  console.log('My favorite color is '+colors[i]);
+}
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  firstName: 'bill',
+  lastName: 'nye',
+  title: 'science guy'
+}
 // Add a `facial-hair` property to `instructor`.
-
+instructor.facialHair = false
```

Comment:
The point of this question was to add a property to this object spelled exactly like `facial-hair`. So really it's asking, "How do I add a property that contains a hyphen to an object?".